instagram/src/App.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import dummyData from './dummy-data.js';
+import PostContainer from './components/PostContainer/PostContainer.js';
+import SearchBar from './components/SearchBar/SearchBar.js';
+import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import {faUserCircle} from '@fortawesome/free-solid-svg-icons';
+import PropType from 'prop-types';
+import PostsPage from './components/PostContainer/PostsPage.js';
+import Authenticate from './components/Authentication/Authenticate.js';
+import Login from './components/Login/Login.js';
+
+
+library.add(faHeart);
+library.add(faUserCircle);
+
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      stateComments: [],
```

Comment:
seems like stateComments, commentInput aren't getting used, so you can probably take them off state